### ISSUE TITLE
10$: disallow newline after 'in'/'out'/'const' modifier in generic parameter list

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1118]
A newline after `in`, `out`, or `const` is disallowed in TypeScript:
```typescript
class C<in
                  T> {
}
```
We should report a nice diagnostic.
